packages/components/src/components/radio-group/radio-group.tsx

```diff
@@ -45,41 +51,47 @@ const RadioGroup = ({
     return (
         <div className={classNames('dc-radio-group', className)}>
             {Array.isArray(children) &&
-                children.map(item => (
-                    <ItemWrapper key={item.props.value} should_wrap_items={should_wrap_items}>
-                        <label
-                            className={classNames('dc-radio-group__item', className, {
-                                'dc-radio-group__item--selected': selected_option === item.props.value,
-                            })}
-                        >
-                            <input
-                                id={item.props.id}
-                                name={name}
-                                className='dc-radio-group__input'
-                                type='radio'
-                                value={item.props.value}
-                                checked={selected_option === item.props.value}
-                                onChange={onChange}
-                                disabled={item.props.disabled}
-                                required={required}
-                            />
-                            <span
-                                className={classNames('dc-radio-group__circle', {
-                                    'dc-radio-group__circle--selected': selected_option === item.props.value,
-                                    'dc-radio-group__circle--disabled': item.props.disabled,
-                                })}
-                            />
-                            <Text
-                                size='xs'
-                                className={classNames('dc-radio-group__label', {
-                                    'dc-radio-group__label--disabled': item.props.disabled,
-                                })}
-                            >
-                                {item.props.label}
-                            </Text>
-                        </label>
-                    </ItemWrapper>
-                ))}
+                children
+                    .filter(item => !item.props.is_hidden)
+                    .map(
+                        item =>
+                            !item.props.is_hidden && (
```

Comment:
We dont need this condition here anymore right?

Reply:
updated